.changelog/9181.txt

```diff
@@ -0,0 +1,6 @@
+```release-note:bug
```

Comment:
should this be new feature or improvement instead of bug?

Reply:
I was talking thru this with Mike and we're just gonna do one entry under deprecation for this PR, then for the additional changes they can be improvements (#9181) or bug fixes (#9182) as appropriate.